Build.proj

```diff
@@ -28,6 +28,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <Npm Condition="'$(Npm)' == ''">npm</Npm>
+    <NuGet Condition="'$(NuGet)' == '' And Exists('$(MSBuildThisFileDirectory)nuget.exe')">$(MSBuildThisFileDirectory)nuget.exe</NuGet>
     <NuGet Condition="'$(NuGet)' == '' And Exists('$(Provisionator_ArtifactCachePath)nuget.exe')">$(Provisionator_ArtifactCachePath)nuget.exe</NuGet>
```

Comment:
Should the order be switched here so we prefer provisionator's nuget.exe?

Reply:
No. If you manually go and drop your own nuget.exe into the repo, you probably really want to use that one?

Reply:
(Also note where I would like to do a second pass at making this nicer)